src/Apps/CollectorProfile/Routes/Saves2/Components/SavesItem.tsx

```diff
@@ -3,60 +3,72 @@ import { FourUpImageLayout } from "Apps/CollectorProfile/Routes/Saves2/Component
 import { StackedImageLayout } from "Apps/CollectorProfile/Routes/Saves2/Components/StackedImageLayout"
 import { FC } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import { RouterLink } from "System/Router/RouterLink"
 import { extractNodes } from "Utils/extractNodes"
 import { SavesItem_item$data } from "__generated__/SavesItem_item.graphql"
 
 interface SavesItemProps {
   isSelected?: boolean
   imagesLayout: "stacked" | "grid"
   item: SavesItem_item$data
-  onClick: () => void
 }
 
-const SavesItem: FC<SavesItemProps> = ({
-  isSelected,
-  imagesLayout,
-  item,
-  onClick,
-}) => {
+const BASE_PATH = "/collector-profile/saves2"
+
+const SavesItem: FC<SavesItemProps> = ({ isSelected, imagesLayout, item }) => {
   const artworkNodes = extractNodes(item.artworksConnection)
   const imageURLs = artworkNodes.map(node => node.image?.url ?? null)
 
+  const getLink = () => {
+    if (item.default) {
+      return BASE_PATH
+    }
+
+    return `${BASE_PATH}/${item.internalID}`
+  }
+
   return (
-    <Flex
-      p={1}
-      width={[138, 222]}
-      height={[188, 272]}
-      border="1px solid"
-      borderRadius={10}
-      flexDirection="column"
-      justifyContent="space-between"
-      borderColor={isSelected ? "brand" : "transparent"}
-      style={isSelected ? { boxShadow: DROP_SHADOW } : undefined}
-      onClick={onClick}
+    <RouterLink
+      to={getLink()}
+      textDecoration="none"
+      aria-current={!!isSelected}
     >
-      {imagesLayout === "stacked" ? (
-        <StackedImageLayout imageURLs={imageURLs} />
-      ) : (
-        <FourUpImageLayout imageURLs={imageURLs} />
-      )}
+      <Flex
+        p={1}
+        width={[138, 222]}
+        height={[188, 272]}
+        border="1px solid"
+        borderRadius={10}
+        flexDirection="column"
+        justifyContent="space-between"
+        borderColor={isSelected ? "brand" : "transparent"}
+        style={isSelected ? { boxShadow: DROP_SHADOW } : undefined}
+      >
+        {imagesLayout === "stacked" ? (
+          <StackedImageLayout imageURLs={imageURLs} />
+        ) : (
+          <FourUpImageLayout imageURLs={imageURLs} />
+        )}
 
-      <Box>
-        <Text variant={["xs", "sm-display"]} overflowEllipsis>
-          {item.name}
-        </Text>
-        <Text variant={["xs", "sm-display"]} color="black60" overflowEllipsis>
-          {`${item.artworksCount} Artworks`}
-        </Text>
-      </Box>
-    </Flex>
+        <Box>
+          <Text variant={["xs", "sm-display"]} overflowEllipsis>
+            {item.name}
+          </Text>
+          <Text variant={["xs", "sm-display"]} color="black60" overflowEllipsis>
+            {`${item.artworksCount} Artworks`}
```

Comment:
**suggestion (non-blocking):** We should make sure to pluralize this properly. I believe [our i18n solution](https://github.com/artsy/force/blob/main/docs/i18n.md) gives us a standard way of doing this now, if/when we adopt that pattern for our UI texts.

For example if [src/System/i18n/locales/en-US/translation.json](https://github.com/artsy/force/blob/main/src/System/i18n/locales/en-US/translation.json) contains:

```js
{
  ...
  "artworkLists": {
    "artworkWithCount_one": "{{count}} artwork",
    "artworkWithCount_other": "{{count}} artworks"
  }
}
```

Then L58 can become:

```js
{t("artworkLists.artworkWithCount", { count: item.artworksCount })}
```

Reply:
Good catch, totally forgot about it. Added changes